.github/workflows/dead-link-checker.yaml

```
@@ -21,22 +21,14 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - uses: actions/cache@v2
+      - name: Setup Node
+        uses: actions/setup-node@v3
         with:
-          path: /tmp/hugo_cache
-          key: ${{ runner.os }}-hugomod-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-hugomod-
+          node-version: 16.16
```

1. Enable cache
2. Use node version from file (https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file)

```suggestion
          node-version-file: '.nvmrc'
          cache: 'npm'
```

---

Ooo I didn't know about `node-version-file` thanks for that.
I've changed it also in another job (qa-checker)